Docs/update calib (#4)
tier4/drs-docs: docs/calibration/index.md

source ./install/setup.bash
```

> [!NOTE]
> The following calibration procedure assumes specific ID and orientation for the target April tag.

> > The tag needs to be equipped to the frame as follows:

Reviewer: ```suggestion
> This exact tag needs to be mounted to the frame in the illustrated orientation:
```

        6. When the number of detected pairs is over the predefined value, the “Save calibration” button will become available. After collecting sufficient data, press the button and save the result into a yaml file. After confirming that the result is correctly saved, close all windows. 
            ![](images/image-20241121-122343.png)

    <details>

>     <summary>If no images are shown on `image_view` window</summary>

Reviewer: ```suggestion
    <summary>If no images are shown in the `image_view` window</summary>
```

>     The possible reason of the issue is time synchronization does not work well for some reason.

Reviewer: ```suggestion
    A possible cause of this issue is poor time synchronization.
```

>     Check the window title of `image_view` window. If the displayed "delay" value is too large like the following picture, the tolerance value can be relaxed by modifing `common/tier4_calibration_views/tier4_calibration_views/image_view_ros_interface.py`.

Reviewer: ```suggestion
    Check the window title of `image_view` window. If the displayed "delay" value is too large, as in the following picture, the tolerance value can be relaxed by modifying `common/tier4_calibration_views/tier4_calibration_views/image_view_ros_interface.py`.
```
